### ISSUE TITLE
VNet template cannot be created/edited by Sunstone (PHYDEV malformed)

### ISSUE THREAD

**Issue by Yenya** [OpenNebula/one#5267]
**Description**
Originally discussed here: https://forum.opennebula.io/t/vnet-template-and-cluster-ids/9253/3

When I try to create a VNet template in Sunstone (5.12.0.3 CE, as oneadmin) it fails with `One or more required fields are missing or malformed`, and the highlighted field is `Physical device`, where I am trying to put the value `eth1`.

When I try to create the template using `onevntemplate create` with the following input file, it works:

```
NAME="VXLAN test"
AUTOMATIC_VLAN_ID="YES"
CLUSTER_IDS="101"
FILTER_MAC_SPOOFING="YES"
GUEST_MTU="1500"
INBOUND_AVG_BW="100000"
OUTBOUND_AVG_BW="100000"
PHYDEV="eth1"
VN_MAD="vxlan"
```

However, any attempt to edit the newly created template in Sunstone fails with the same error message and the same highlited field `Physical device`.

So I guess the validation of the `Physical device` field in Sunstone is broken.

**To Reproduce**
See above.

**Expected behavior**
Created/edited VNet template in Sunstone.

**Details**
 - Affected Component: Sunstone
 - Hypervisor: KVM
 - Version: 5.12.0.3 CE (5.12.0.1 was OK), CentOS 7

**Additional context**
I have the `eth1` interface active only on the nodes of cluster 101, not on the server where main processes of OpenNebula (oned/sunstone/...) are running.

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by FrederickBor** [comment]
Hi, I tried to reproduce this on 5.12.0.3 CE but I don't have the same result that you have, can you provide more information?

**Comment by Yenya** [comment]
Frederick, now I tried to reproduce it myself, and I feel pretty stupid, because I cannot reproduce the problem :-/

When I wrote the issue text and even before during writing to forum.o.io it was definitely broken. Thinking about what could have changed since then, I have restarted my browser and re-login as oneadmin. Could it be that I had some old javascripts or something cached during the upgrade from 5.12.0.1 to 5.12.0.3?

Anyway, thanks for your time and sorry for the noise.
